en/docs/reference/connectors/pulsar-connector/pulsar-connector-example.md

```diff
@@ -0,0 +1,260 @@
+
+### Setup Environment for Apache Pulsar Connector
+
+To configure the Apache Pulsar connector with Apache Pulsar version 4.0.4, copy the following client libraries to the `<MI_HOME>/lib` directory.
+
+* [pulsar-client-4.0.4.wso2v1.jar](https://mvnrepository.com/artifact/org.apache.kafka/kafka_2.12/2.8.2)
+* [pulsar-client-original-4.0.4.jar](https://repo1.maven.org/maven2/org/apache/pulsar/pulsar-client-original/4.0.4/pulsar-client-original-4.0.4.jar)
+* [pulsar-common-4.0.4.jar](https://repo1.maven.org/maven2/org/apache/pulsar/pulsar-common/4.0.4/pulsar-common-4.0.4.jar)
+* [bookkeeper-common-allocator-4.16.6.jar](https://repo1.maven.org/maven2/org/apache/bookkeeper/bookkeeper-common-allocator/4.16.6/bookkeeper-common-allocator-4.16.6.jar)
+* [netty-codec-dns-4.1.119.Final.jar](https://repo1.maven.org/maven2/io/netty/netty-codec-dns/4.1.119.Final/netty-codec-dns-4.1.119.Final.jar)
+* [netty-resolver-dns-4.1.119.Final.jar](https://repo1.maven.org/maven2/io/netty/netty-resolver-dns/4.1.119.Final/netty-resolver-dns-4.1.119.Final.jar)
+* [netty-transport-classes-epoll-4.1.119.Final.jar](https://repo1.maven.org/maven2/io/netty/netty-transport-classes-epoll/4.1.119.Final/netty-transport-classes-epoll-4.1.119.Final.jar)
+* [netty-incubator-transport-classes-io_uring-0.0.26.Final.jar](https://repo1.maven.org/maven2/io/netty/incubator/netty-incubator-transport-classes-io_uring/0.0.26.Final/netty-incubator-transport-classes-io_uring-0.0.26.Final.jar)
+
+## Set up the integration project
+
+Follow these steps to set up the Integration Project using the WSO2 Micro Integrator Visual Studio Code extension.
+
+### Create a new project
+
+Follow the steps in the [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the **WSO2 MI** and create the integration project with the **Project Name** as follows:
+
+<img src="{{base_path}}/assets/img/integrate/connectors/pulsar/CreateProject.png" title="Creating a new Project" width="800" alt="Creating a new Project"/>
+
+### Create the integration logic
+
+#### Create the API
+
+1. Under the **Create an integration** section, select **API** to create a new REST API.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/CreateAPI1.png" title="Creating a new API" width="800" alt="Creating a new API"/>
+
+2. Then, enter the API name as `PublishMessage` and the Context as `/publishMessage` and click **Create**.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/CreateAPI.png" title="Creating a new API" width="800" alt="Creating a new API"/>
+
+3. Select the newly created `PublishMessage` API and click the `edit` button to change the API method.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/UpdateResource1.png" title="Adding the API resource step 1." width="900" alt="Adding the API resource step 1."/>
+
+4. Then select the `POST` method and click `Update`.
```

Comment:
The code style is to use **bold text** to refer UI elements. Please check other places as well.

```suggestion
4. Then select the `POST` method and click **Update**.
```